<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSwitchBar</class>
 <widget class="QWidget" name="PageSwitchBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>60</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_main_menu">
       <property name="text">
        <string>Main Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_pf_ablation">
       <property name="text">
        <string>Pf Ablation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_rf_ablation">
       <property name="text">
        <string>Rf Ablation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_rf_puncture">
       <property name="text">
        <string>Rf Puncture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_post">
       <property name="text">
        <string>Post</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
